format/spec.md

```diff
@@ -392,7 +392,7 @@ In v3 and later, an Iceberg table can track row lineage fields for all newly cre
 
 These fields are assigned and updated by inheritance because the commit sequence number and starting row ID are not assigned until the snapshot is successfully committed. Inheritance is used to allow writing data and manifest files before values are known so that it is not necessary to rewrite data and manifest files when an optimistic commit is retried.
 
-When row lineage is enabled, new snapshots cannot include [Equality Deletes](#equality-delete-files). Row lineage is incompatible with equality deletes because lineage values must be maintained, but equality deletes are used to avoid reading existing data before writing changes.
+Row lineage does not track updates for rows updated because of [Equality Deletes](#equality-delete-files). Rows updated via an equality delete are always treated as if the row was completely removed and a unique new row was created.
```

Comment:
wonder if this is a bit more clear
```
Row lineage does not track lineage for rows updated via [Equality Deletes](#equality-delete-files), because equality deletes avoid reading existing data before writing changes. A equality delete should always be treated as if the existing row was completely removed and a unique new row was added.
```

Reply:
[optional] May be rephrase the last line in suggestion above : An update via equality delete should
always be treated as if the existing row was completely removed and a unique new row was added.

Reply:
Maybe it is too technical, but equality delete doesn't have to do anything with row lineage. The engines are to blame which doesn't populate the old rowId in the new version of the row.